cobbler.spec

```diff
@@ -306,6 +299,7 @@ fi
 %post
 %{py3_bytecompile_post %{name}}
 %{systemd_post cobblerd.service}
+%{apache2_module_post proxy_http}
```

Comment:
Please add `BuildRequires: apache2-deb-macros` in a debbuild vendor conditional

Reply:
Fixed.